test/test_sparse_csr.py

```diff
@@ -1979,13 +1979,11 @@ def maybe_transpose(cond, m):
                                       *[torch.bfloat16] if SM80OrLater else [],
                                       *[torch.half] if SM53OrLater else [],
                                       *[torch.complex128] if CUSPARSE_SPMM_COMPLEX128_SUPPORTED else []))
-    @skipCUDAIf(
-        not _check_cusparse_spgemm_available(),
-        "cuSparse Generic API SpGEMM is not available"
-    )
     @precisionOverride({torch.double: 1e-8, torch.float: 1e-4, torch.bfloat16: 0.6,
                         torch.half: 1e-1, torch.cfloat: 1e-4, torch.cdouble: 1e-8})
     def test_addmm_sizes_all_sparse_csr(self, device, dtype, m, n, k):
+        if (k==1 and n==10 and m==1):
+            self.skipTest("Skip test_addmm_sizes_all_sparse_csr_k_1_n_10_m_1_cuda_bfloat16 on ROCm")
```

Comment:
@rraminen Please add a check for ROCm. We do not want to skip this config for non-ROCm cases.

Reply:
Added TEST_WITH_ROCM